bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -6178,27 +6168,26 @@ public BlackboardArtifact getArtifactById(long id) throws TskCoreException {
 	 *                          core and file could not be queried
 	 */
 	public BlackboardArtifact getArtifactByArtifactId(long id) throws TskCoreException {
-		CaseDbConnection connection = null;
-		ResultSet rs = null;
+		String query = "SELECT artifact_type_id, artifact_obj_id WHERE artifact_id = " + id;
```

Comment:
Can this method be deprecated with a referral to an equivalent Blackboard method?